live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -76,3 +76,10 @@
   delay: 30
   register: result
   until: result is succeeded
+
+#
+# CRA on Linux requires that the engine UUID is stored in a file
+#
+- apt:
+    name: 'dmidecode'
+- shell: dmidecode -s system-uuid | awk '{print tolower($0)}' > /etc/engine-uuid
```

Comment:
we're currently running:
```
192 - command: /opt/delphix/server/bin/register/generate_secret -q
```
via the app-gate ansible role.. should we really be adding this logic, rather than relying on the app-gate ansible role?

Reply:
I just got rid of the need for uuid file altogether in http://reviews.delphix.com/r/50879/, and I'll remove its generation in `generate_secret` next.